ansys/dpf/core/examples/examples.py

```diff
@@ -19,3 +19,4 @@
 msup_transient = os.path.join(_module_path, "msup_transient_plate1.rst")
 simple_cyclic = os.path.join(_module_path, "file_cyclic.rst")
 distributed_msup_folder = os.path.join(_module_path, 'msup_distributed')
```

Comment:
Thanks for this work @ayush-kumar-423, it's great.
We try to have as few result files as possible in this package to not slow down the pip installation. Could you please add your file to this (https://github.com/pyansys/example-data/tree/master/result_files) repo et add a function in examples/download.py to download it?

Reply:
Done!